### PR TITLE
[Japneet | Procheta] Add. select API workflow

### PR DESCRIPTION
**Pull request by japneetkaurbhatia**

### CODE REVIEW

File: src/api/Api.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
-import fetch from 'node-fetch';
```
Comment (hariharanweb) [inline src/api/Api.js]: Run lint

File: src/services/LoggingService.js [hunk 2]
```diff
@@ -4,7 +4,9 @@ import log4js from 'log4js';
 dotenv.config();
 
 const getLogger = (key) => {
-  const logger = key ? log4js.getLogger(`BPP-${key}`) : log4js.getLogger('GatewayRegistry');
+  const logger = key
+    ? log4js.getLogger(`BPP-${key}`)
+    : log4js.getLogger('GatewayRegistry');
```
Comment (hariharanweb) [inline src/services/LoggingService.js]: Make this BPP; Its a existing copy/paste bug;
log4js.getLogger('GatewayRegistry');
should be log4js.getLogger('BPP')

File: src/services/FakeOnSelectResponse.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        },
+        "collected_by": "BPP",
+        "type": "ON-FULFILLMENT"
+      },
+      "items": [
+        {
+          "rating": null,
+          "offer_id": null,
+          "category_id": "DRIVER_OFFER",
+          "./komn/driver_name": "VENKATESH",
```
Comment (hariharanweb) [inline src/services/FakeOnSelectResponse.json]: Remove this non-schema items

File: src/api/Api.js (hunk 1 above)
Comment (prochetab1999) [reply src/api/Api.js]: Updated

File: src/services/LoggingService.js (hunk 2 above)
Comment (prochetab1999) [reply src/services/LoggingService.js]: Updated

File: src/services/FakeOnSelectResponse.json (hunk 3 above)
Comment (prochetab1999) [reply src/services/FakeOnSelectResponse.json]: Will be updated in the new json

**Pull request merged**